### PR TITLE
OpenCV3.4 Python update

### PR DESCRIPTION
**Pull request by YacobBY**
Updated python code for OpenCV3.4, and readme file for clearer instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,43 +1,15 @@
-# Human detection and Tracking
```
Comment (arpit1997) [inline README.md]: why is title removed from README?

File: README.md [hunk 2]
```diff
@@ -46,48 +18,3 @@ Example- for our directory structure it is:
 ```sh
  python3 main.py /video 
 ```
-* To compile the C++ version of the code with openCV the command is:
```
Comment (arpit1997) [inline README.md]: why this whole paragraph is stripped? @YacobBY Can you please explain? the README looks way too much small with little to no information about the project.

Comment (YacobBY) [reply README.md]: That's a mistake should be put back, is for my personal version.

Comment (YacobBY) [reply README.md]: Didn't realize it would commit changes I made after submitting the version to you

File: main.py [hunk 3]
```diff
@@ -13,171 +14,172 @@
...
+    temp = 0
+    for c in cnts:
+        # if the contour is too small, ignore it
+        if cv2.contourArea(c) > min_area:
+            temp = 1
+    return temp
+
 
 if __name__ == '__main__':
-	"""
```
Comment (arpit1997) [inline main.py:125]: what's the improvement in this refactor?

File: README.md (hunk 2 above)
Comment (arpit1997) [reply README.md]: please resolve this issue if its a mistake

File: main.py (hunk 3 above)
Comment (YacobBY) [reply main.py:125]: It's the standard IntelliJ/Pycharm refactor. Personally I think it's more readable like this, but the most important thing is the lbph creation which got changed in OpenCV3.4.

Also might be handy to set the standard path to the video directory in the argparse.

**Comment by YacobBY** [comment]
Readme should be fixed now

**Comment by prsntmaurya** [comment]
Hi, 
This is already resolved in issue #25, there is no need to update the code.

**Comment by YacobBY** [comment]
No you have not solved the issue. People have to manually change the broken code every time. If you fix your code you don't have to deal with people asking the same question over and over.

**Comment by arpit1997** [comment]
@YacobBY Sorry about closing the PR. Merged it now. Thanks for your valuable contribution

**Pull request merged**